packages/dashboard/src/features/compute/constants.ts

```
@@ -30,10 +30,29 @@ export const REGIONS = [
+ * connect with the protocol-native client (redis-cli, psql, etc.).
  */
-export function getReachableUrl(service: ServiceSchema): string | null {
-  return service.endpointUrl;
+export interface ServiceEndpoint {
+  display: string;
+  /** null for TCP — browsers cannot navigate to raw TCP endpoints. */
+  href: string | null;
+}
+
+export function getReachableUrl(service: ServiceSchema): ServiceEndpoint | null {
+  if (!service.endpointUrl) return null;
```

ESLint

[error] Expected { after 'if' condition.

(curly)